### PR TITLE
VEN-1181: resending of cancelled orders. Set order status back to "wa…

### PR DESCRIPTION
**Pull request by haavikko**
…iting" when resending

### Closes :no_good_woman:
**[VEN-1181](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1181):**

### CODE REVIEW

**Review by klemmari1 — APPROVED** [review]
Looks good to me! :ship:

File: payments/schema/mutations.py [hunk 1]
```diff
@@ -640,6 +640,19 @@ class Input:
...
+            order.set_status(
+                OrderStatus.WAITING,
+                comment=f"{today()}: {_('Cleanup the invoice to attempt resending')}\n",
+            )
+        elif order.status == OrderStatus.CANCELLED:
+            order.set_status(
+                OrderStatus.WAITING,
+                comment=f"{today()}: {_('Resend cancelled invoice')}\n",
+            )
+
```
Comment (a-rmz) [inline payments/schema/mutations.py:655]: This should go to `payments/utils.py` to make it a reusable util :)

Comment (haavikko) [reply payments/schema/mutations.py]: fixed

**Review by a-rmz — APPROVED** [review]

**Pull request merged**